src/components/AiMarketplace/MainSection/Filter/StyledExpansionPanel.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component, Fragment } from "react";
+import ExpansionPanel from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import Typography from "@material-ui/core/Typography";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import { withStyles } from "@material-ui/styles";
+
+import { useStyles } from "./styles";
+
+class StyledExpansionPanel extends Component {
+    state = {
+        expansionData: [
+            {
+                title: "Categories",
+                items: [
+                    {
+                        title: "Text Analysis",
+                        count: "25",
+                    },
+                    {
+                        title: "Computer Vision",
+                        count: "21",
+                    },
+                    {
+                        title: "Deep Learning",
+                        count: "17",
+                    },
+                ],
+            },
+            {
+                title: "Price",
+                items: [
+                    {
+                        title: "Machine Learning",
+                        count: "29",
+                    },
+                    {
+                        title: "Sentiments Analysis",
+                        count: "18",
+                    },
+                    {
+                        title: "Microservices",
+                        count: "18",
+                    },
+                ],
+            },
+            {
+                title: "Vendors",
+                items: [
+                    {
+                        title: "Time Series",
+                        count: "17",
+                    },
+                    {
+                        title: "Utilities",
+                        count: "18",
+                    },
+                ],
+            },
+        ],
+    };
+
+    render() {
+        const { classes } = this.props;
+        const { expansionData } = this.state;
+
+        return (
+            <Fragment>
+                {expansionData.map(val => (
+                    <ExpansionPanel className={classes.filterExpansionPanel}>
```

Comment:
Key missing.

Reply:
fixed in the [commit](https://github.com/singnet/refactored-dapp/pull/2/commits/09c13f188b76a84c0732bfb7f73cc9adadc102fa) and [commit](https://github.com/singnet/refactored-dapp/pull/2/commits/930dfb3c8818b72defe2aa1e87f28cbfbe67f86b)